datafusion/core/src/physical_plan/aggregates/row_hash.rs

```diff
@@ -110,12 +111,15 @@ impl GroupedHashAggregateStreamV2 {
         // The expressions to evaluate the batch, one vec of expressions per aggregation.
         // Assume create_schema() always put group columns in front of aggr columns, we set
         // col_idx_base to group expression count.
-        let aggregate_expressions =
-            aggregates::aggregate_expressions(&aggr_expr, &mode, group_expr.len())?;
+        let aggregate_expressions = aggregates::aggregate_expressions(
```

Comment:
FYI @yjshen  -- it would be really nice to try and consolidate `row_hash` and `hash` -- filed https://github.com/apache/arrow-datafusion/issues/2723 to track 👍